torch/_inductor/fx_passes/reinplace.py

```diff
@@ -460,20 +462,36 @@ def is_meta_only_user(node):
                 return True
         return False
 
+    def _overlap(ls) -> bool:
+        try:
+            return len(compute_overlapping_tensors(ls)) != 0
+        except GuardOnDataDependentSymNode:
+            # If we fail with data dependent error we assume they all overlap.
+            return True
```

Comment:
nit: "_any_tensors_overlap_conservative"?

Also this doesn't need to be a method on the class, right?

Reply:
it does not i can move it more up outside

Reply:
made it local